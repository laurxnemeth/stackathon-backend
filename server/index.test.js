// const app = require('./index.js');
// const supertest = require('supertest');
// const request = supertest(app);


// test('Hello world works', async (done) => {
// 	const response = await request(app.callback()).get('/');
// 	expect(response.status).toBe(200);
// 	done()
// });