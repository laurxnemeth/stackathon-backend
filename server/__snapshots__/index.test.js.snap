// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Hello world works 1`] = `"{\\"10\\":true,\\"39\\":true,\\"'ll\\":true,\\"'tis\\":true,\\"'twas\\":true,\\"'ve\\":true,\\"a\\":true,\\"a's\\":true,\\"able\\":true,\\"ableabout\\":true,\\"about\\":true,\\"above\\":true,\\"abroad\\":true,\\"abst\\":true,\\"accordance\\":true,\\"according\\":true,\\"accordingly\\":true,\\"across\\":true,\\"act\\":true,\\"actually\\":true,\\"ad\\":true,\\"added\\":true,\\"adj\\":true,\\"adopted\\":true,\\"ae\\":true,\\"af\\":true,\\"affected\\":true,\\"affecting\\":true,\\"affects\\":true,\\"after\\":true,\\"afterwards\\":true,\\"ag\\":true,\\"again\\":true,\\"against\\":true,\\"ago\\":true,\\"ah\\":true,\\"ahead\\":true,\\"ai\\":true,\\"ain't\\":true,\\"aint\\":true,\\"al\\":true,\\"all\\":true,\\"allow\\":true,\\"allows\\":true,\\"almost\\":true,\\"alone\\":true,\\"along\\":true,\\"alongside\\":true,\\"already\\":true,\\"also\\":true,\\"although\\":true,\\"always\\":true,\\"am\\":true,\\"amid\\":true,\\"amidst\\":true,\\"among\\":true,\\"amongst\\":true,\\"amoungst\\":true,\\"amount\\":true,\\"an\\":true,\\"and\\":true,\\"announce\\":true,\\"another\\":true,\\"any\\":true,\\"anybody\\":true,\\"anyhow\\":true,\\"anymore\\":true,\\"anything\\":true,\\"anyway\\":true,\\"anyways\\":true,\\"anywhere\\":true,\\"ao\\":true,\\"apart\\":true,\\"apparently\\":true,\\"appear\\":true,\\"appreciate\\":true,\\"appropriate\\":true,\\"approximately\\":true,\\"aq\\":true,\\"ar\\":true,\\"are\\":true,\\"area\\":true,\\"areas\\":true,\\"aren\\":true,\\"aren't\\":true,\\"arent\\":true,\\"arise\\":true,\\"around\\":true,\\"arpa\\":true,\\"as\\":true,\\"aside\\":true,\\"ask\\":true,\\"asked\\":true,\\"asking\\":true,\\"asks\\":true,\\"associated\\":true,\\"at\\":true,\\"au\\":true,\\"auth\\":true,\\"available\\":true,\\"aw\\":true,\\"away\\":true,\\"awfully\\":true,\\"az\\":true,\\"b\\":true,\\"ba\\":true,\\"back\\":true,\\"backed\\":true,\\"backing\\":true,\\"backs\\":true,\\"backward\\":true,\\"backwards\\":true,\\"bb\\":true,\\"bd\\":true,\\"be\\":true,\\"became\\":true,\\"because\\":true,\\"become\\":true,\\"becomes\\":true,\\"becoming\\":true,\\"been\\":true,\\"before\\":true,\\"beforehand\\":true,\\"began\\":true,\\"begin\\":true,\\"beginning\\":true,\\"beginnings\\":true,\\"begins\\":true,\\"behind\\":true,\\"being\\":true,\\"beings\\":true,\\"believe\\":true,\\"below\\":true,\\"beside\\":true,\\"besides\\":true,\\"best\\":true,\\"better\\":true,\\"between\\":true,\\"beyond\\":true,\\"bf\\":true,\\"bg\\":true,\\"bh\\":true,\\"bi\\":true,\\"big\\":true,\\"bill\\":true,\\"billion\\":true,\\"biol\\":true,\\"bj\\":true,\\"bm\\":true,\\"bn\\":true,\\"bo\\":true,\\"both\\":true,\\"bottom\\":true,\\"br\\":true,\\"brief\\":true,\\"briefly\\":true,\\"bs\\":true,\\"bt\\":true,\\"but\\":true,\\"buy\\":true,\\"bv\\":true,\\"bw\\":true,\\"by\\":true,\\"bz\\":true,\\"c\\":true,\\"c'mon\\":true,\\"c's\\":true,\\"ca\\":true,\\"call\\":true,\\"came\\":true,\\"can\\":true,\\"can't\\":true,\\"cannot\\":true,\\"cant\\":true,\\"caption\\":true,\\"case\\":true,\\"cases\\":true,\\"cause\\":true,\\"causes\\":true,\\"cc\\":true,\\"cd\\":true,\\"certain\\":true,\\"certainly\\":true,\\"cf\\":true,\\"cg\\":true,\\"ch\\":true,\\"changes\\":true,\\"ci\\":true,\\"ck\\":true,\\"cl\\":true,\\"clear\\":true,\\"clearly\\":true,\\"click\\":true,\\"cm\\":true,\\"cmon\\":true,\\"cn\\":true,\\"co\\":true,\\"co.\\":true,\\"com\\":true,\\"come\\":true,\\"comes\\":true,\\"computer\\":true,\\"con\\":true,\\"concerning\\":true,\\"copy\\":true,\\"corresponding\\":true,\\"could\\":true,\\"could've\\":true,\\"couldn\\":true,\\"couldn't\\":true,\\"couldnt\\":true,\\"course\\":true,\\"cr\\":true,\\"cry\\":true,\\"cs\\":true,\\"cu\\":true,\\"currently\\":true,\\"cv\\":true,\\"cx\\":true,\\"cy\\":true,\\"cz\\":true,\\"d\\":true,\\"dare\\":true,\\"daren't\\":true,\\"darent\\":true,\\"date\\":true,\\"de\\":true,\\"dear\\":true,\\"definitely\\":true,\\"describe\\":true,\\"described\\":true,\\"despite\\":true,\\"detail\\":true,\\"did\\":true,\\"didn\\":true,\\"didn't\\":true,\\"didnt\\":true,\\"differ\\":true,\\"different\\":true,\\"differently\\":true,\\"directly\\":true,\\"dj\\":true,\\"dk\\":true,\\"dm\\":true,\\"do\\":true,\\"does\\":true,\\"doesn\\":true,\\"doesn't\\":true,\\"doesnt\\":true,\\"doing\\":true,\\"don\\":true,\\"don't\\":true,\\"done\\":true,\\"dont\\":true,\\"doubtful\\":true,\\"down\\":true,\\"downed\\":true,\\"downing\\":true,\\"downs\\":true,\\"downwards\\":true,\\"due\\":true,\\"during\\":true,\\"dz\\":true,\\"e\\":true,\\"each\\":true,\\"early\\":true,\\"ec\\":true,\\"ed\\":true,\\"edu\\":true,\\"ee\\":true,\\"effect\\":true,\\"eg\\":true,\\"eh\\":true,\\"eight\\":true,\\"eighty\\":true,\\"either\\":true,\\"eleven\\":true,\\"else\\":true,\\"elsewhere\\":true,\\"empty\\":true,\\"end\\":true,\\"ended\\":true,\\"ending\\":true,\\"ends\\":true,\\"enough\\":true,\\"er\\":true,\\"es\\":true,\\"et\\":true,\\"et-al\\":true,\\"etc\\":true,\\"even\\":true,\\"evenly\\":true,\\"ever\\":true,\\"evermore\\":true,\\"every\\":true,\\"everybody\\":true,\\"everyone\\":true,\\"everything\\":true,\\"everywhere\\":true,\\"ex\\":true,\\"exactly\\":true,\\"example\\":true,\\"except\\":true,\\"f\\":true,\\"face\\":true,\\"faces\\":true,\\"fact\\":true,\\"facts\\":true,\\"far\\":true,\\"farther\\":true,\\"felt\\":true,\\"few\\":true,\\"fewer\\":true,\\"ff\\":true,\\"fi\\":true,\\"fifteen\\":true,\\"fifth\\":true,\\"fifty\\":true,\\"fify\\":true,\\"fill\\":true,\\"find\\":true,\\"finds\\":true,\\"fire\\":true,\\"first\\":true,\\"five\\":true,\\"fix\\":true,\\"fj\\":true,\\"fk\\":true,\\"fm\\":true,\\"fo\\":true,\\"followed\\":true,\\"following\\":true,\\"follows\\":true,\\"for\\":true,\\"forever\\":true,\\"former\\":true,\\"forth\\":true,\\"forty\\":true,\\"forward\\":true,\\"found\\":true,\\"four\\":true,\\"fr\\":true,\\"free\\":true,\\"from\\":true,\\"front\\":true,\\"full\\":true,\\"fully\\":true,\\"further\\":true,\\"furthered\\":true,\\"furthering\\":true,\\"furthermore\\":true,\\"furthers\\":true,\\"fx\\":true,\\"g\\":true,\\"ga\\":true,\\"gave\\":true,\\"gb\\":true,\\"gd\\":true,\\"ge\\":true,\\"general\\":true,\\"get\\":true,\\"gets\\":true,\\"getting\\":true,\\"gf\\":true,\\"gg\\":true,\\"gh\\":true,\\"gi\\":true,\\"give\\":true,\\"given\\":true,\\"gives\\":true,\\"giving\\":true,\\"gl\\":true,\\"gm\\":true,\\"gmt\\":true,\\"gn\\":true,\\"go\\":true,\\"goes\\":true,\\"going\\":true,\\"gone\\":true,\\"good\\":true,\\"goods\\":true,\\"got\\":true,\\"gotten\\":true,\\"gov\\":true,\\"gp\\":true,\\"gq\\":true,\\"gr\\":true,\\"great\\":true,\\"greater\\":true,\\"greatest\\":true,\\"greetings\\":true,\\"grouped\\":true,\\"grouping\\":true,\\"groups\\":true,\\"gs\\":true,\\"gt\\":true,\\"gu\\":true,\\"gw\\":true,\\"gy\\":true,\\"h\\":true,\\"had\\":true,\\"hadn't\\":true,\\"hadnt\\":true,\\"half\\":true,\\"happens\\":true,\\"hardly\\":true,\\"has\\":true,\\"hasn\\":true,\\"hasn't\\":true,\\"hasnt\\":true,\\"have\\":true,\\"haven\\":true,\\"haven't\\":true,\\"havent\\":true,\\"having\\":true,\\"he\\":true,\\"he'd\\":true,\\"he'll\\":true,\\"he's\\":true,\\"hed\\":true,\\"hell\\":true,\\"hello\\":true,\\"help\\":true,\\"hence\\":true,\\"her\\":true,\\"here\\":true,\\"here's\\":true,\\"hereafter\\":true,\\"hereby\\":true,\\"herein\\":true,\\"heres\\":true,\\"hereupon\\":true,\\"hers\\":true,\\"herself\\":true,\\"herse”\\":true,\\"hes\\":true,\\"hi\\":true,\\"hid\\":true,\\"high\\":true,\\"higher\\":true,\\"highest\\":true,\\"him\\":true,\\"himself\\":true,\\"himse”\\":true,\\"his\\":true,\\"hither\\":true,\\"hk\\":true,\\"hm\\":true,\\"hn\\":true,\\"home\\":true,\\"homepage\\":true,\\"hopefully\\":true,\\"how\\":true,\\"how'd\\":true,\\"how'll\\":true,\\"how's\\":true,\\"howbeit\\":true,\\"however\\":true,\\"hr\\":true,\\"ht\\":true,\\"htm\\":true,\\"html\\":true,\\"http\\":true,\\"hu\\":true,\\"hundred\\":true,\\"i\\":true,\\"i'd\\":true,\\"i'll\\":true,\\"i'm\\":true,\\"i've\\":true,\\"i.e.\\":true,\\"id\\":true,\\"ie\\":true,\\"if\\":true,\\"ignored\\":true,\\"ii\\":true,\\"il\\":true,\\"ill\\":true,\\"im\\":true,\\"immediate\\":true,\\"immediately\\":true,\\"importance\\":true,\\"important\\":true,\\"in\\":true,\\"inasmuch\\":true,\\"inc\\":true,\\"inc.\\":true,\\"indeed\\":true,\\"index\\":true,\\"indicate\\":true,\\"indicated\\":true,\\"indicates\\":true,\\"information\\":true,\\"inner\\":true,\\"inside\\":true,\\"insofar\\":true,\\"instead\\":true,\\"int\\":true,\\"interest\\":true,\\"interested\\":true,\\"interesting\\":true,\\"interests\\":true,\\"into\\":true,\\"inward\\":true,\\"io\\":true,\\"iq\\":true,\\"ir\\":true,\\"is\\":true,\\"isn\\":true,\\"isn't\\":true,\\"isnt\\":true,\\"it\\":true,\\"it'd\\":true,\\"it'll\\":true,\\"it's\\":true,\\"itd\\":true,\\"itll\\":true,\\"its\\":true,\\"itself\\":true,\\"itse”\\":true,\\"ive\\":true,\\"j\\":true,\\"je\\":true,\\"jm\\":true,\\"jo\\":true,\\"join\\":true,\\"jp\\":true,\\"just\\":true,\\"k\\":true,\\"ke\\":true,\\"keep\\":true,\\"keeps\\":true,\\"kept\\":true,\\"keys\\":true,\\"kg\\":true,\\"kh\\":true,\\"ki\\":true,\\"kind\\":true,\\"km\\":true,\\"kn\\":true,\\"knew\\":true,\\"know\\":true,\\"known\\":true,\\"knows\\":true,\\"kp\\":true,\\"kr\\":true,\\"kw\\":true,\\"ky\\":true,\\"kz\\":true,\\"l\\":true,\\"la\\":true,\\"large\\":true,\\"largely\\":true,\\"last\\":true,\\"lately\\":true,\\"later\\":true,\\"latest\\":true,\\"latter\\":true,\\"latterly\\":true,\\"lb\\":true,\\"lc\\":true,\\"least\\":true,\\"length\\":true,\\"less\\":true,\\"lest\\":true,\\"let\\":true,\\"let's\\":true,\\"lets\\":true,\\"li\\":true,\\"like\\":true,\\"liked\\":true,\\"likely\\":true,\\"likewise\\":true,\\"line\\":true,\\"little\\":true,\\"lk\\":true,\\"ll\\":true,\\"long\\":true,\\"longer\\":true,\\"longest\\":true,\\"look\\":true,\\"looking\\":true,\\"looks\\":true,\\"low\\":true,\\"lower\\":true,\\"lr\\":true,\\"ls\\":true,\\"lt\\":true,\\"ltd\\":true,\\"lu\\":true,\\"lv\\":true,\\"ly\\":true,\\"m\\":true,\\"ma\\":true,\\"make\\":true,\\"makes\\":true,\\"making\\":true,\\"man\\":true,\\"many\\":true,\\"may\\":true,\\"maybe\\":true,\\"mayn't\\":true,\\"maynt\\":true,\\"mc\\":true,\\"md\\":true,\\"me\\":true,\\"mean\\":true,\\"means\\":true,\\"meantime\\":true,\\"meanwhile\\":true,\\"member\\":true,\\"members\\":true,\\"mg\\":true,\\"mh\\":true,\\"microsoft\\":true,\\"might\\":true,\\"might've\\":true,\\"mightn't\\":true,\\"mightnt\\":true,\\"mil\\":true,\\"mill\\":true,\\"million\\":true,\\"mine\\":true,\\"minus\\":true,\\"miss\\":true,\\"mk\\":true,\\"ml\\":true,\\"mm\\":true,\\"mn\\":true,\\"mo\\":true,\\"more\\":true,\\"moreover\\":true,\\"most\\":true,\\"move\\":true,\\"mp\\":true,\\"mq\\":true,\\"mr\\":true,\\"mrs\\":true,\\"ms\\":true,\\"msie\\":true,\\"mt\\":true,\\"mu\\":true,\\"much\\":true,\\"mug\\":true,\\"must\\":true,\\"must've\\":true,\\"mustn't\\":true,\\"mustnt\\":true,\\"mv\\":true,\\"mw\\":true,\\"mx\\":true,\\"my\\":true,\\"myself\\":true,\\"myse”\\":true,\\"mz\\":true,\\"n\\":true,\\"na\\":true,\\"name\\":true,\\"nay\\":true,\\"nc\\":true,\\"nd\\":true,\\"ne\\":true,\\"near\\":true,\\"need\\":true,\\"needed\\":true,\\"needing\\":true,\\"needn't\\":true,\\"neednt\\":true,\\"needs\\":true,\\"neither\\":true,\\"net\\":true,\\"netscape\\":true,\\"never\\":true,\\"neverf\\":true,\\"neverless\\":true,\\"nevertheless\\":true,\\"new\\":true,\\"newer\\":true,\\"newest\\":true,\\"next\\":true,\\"nf\\":true,\\"ng\\":true,\\"ni\\":true,\\"nine\\":true,\\"ninety\\":true,\\"nl\\":true,\\"no\\":true,\\"no-one\\":true,\\"nobody\\":true,\\"non\\":true,\\"none\\":true,\\"nonetheless\\":true,\\"noone\\":true,\\"nor\\":true,\\"normally\\":true,\\"nos\\":true,\\"not\\":true,\\"noted\\":true,\\"nothing\\":true,\\"notwithstanding\\":true,\\"novel\\":true,\\"now\\":true,\\"nowhere\\":true,\\"np\\":true,\\"nr\\":true,\\"nu\\":true,\\"null\\":true,\\"nz\\":true,\\"o\\":true,\\"obtain\\":true,\\"obtained\\":true,\\"obviously\\":true,\\"of\\":true,\\"off\\":true,\\"often\\":true,\\"oh\\":true,\\"ok\\":true,\\"okay\\":true,\\"old\\":true,\\"older\\":true,\\"oldest\\":true,\\"om\\":true,\\"omitted\\":true,\\"on\\":true,\\"once\\":true,\\"one\\":true,\\"one's\\":true,\\"ones\\":true,\\"only\\":true,\\"onto\\":true,\\"open\\":true,\\"opened\\":true,\\"opening\\":true,\\"opens\\":true,\\"opposite\\":true,\\"or\\":true,\\"ord\\":true,\\"order\\":true,\\"ordered\\":true,\\"ordering\\":true,\\"orders\\":true,\\"org\\":true,\\"other\\":true,\\"others\\":true,\\"otherwise\\":true,\\"ought\\":true,\\"oughtn't\\":true,\\"oughtnt\\":true,\\"our\\":true,\\"ours\\":true,\\"ourselves\\":true,\\"out\\":true,\\"outside\\":true,\\"over\\":true,\\"overall\\":true,\\"owing\\":true,\\"own\\":true,\\"p\\":true,\\"pa\\":true,\\"page\\":true,\\"pages\\":true,\\"part\\":true,\\"parted\\":true,\\"particular\\":true,\\"parting\\":true,\\"parts\\":true,\\"past\\":true,\\"pe\\":true,\\"per\\":true,\\"perhaps\\":true,\\"pf\\":true,\\"pg\\":true,\\"ph\\":true,\\"pk\\":true,\\"pl\\":true,\\"place\\":true,\\"placed\\":true,\\"places\\":true,\\"please\\":true,\\"plus\\":true,\\"pm\\":true,\\"pmid\\":true,\\"pn\\":true,\\"point\\":true,\\"pointed\\":true,\\"pointing\\":true,\\"points\\":true,\\"possible\\":true,\\"pp\\":true,\\"pr\\":true,\\"present\\":true,\\"presented\\":true,\\"presenting\\":true,\\"presents\\":true,\\"problem\\":true,\\"problems\\":true,\\"proud\\":true,\\"provided\\":true,\\"provides\\":true,\\"pt\\":true,\\"put\\":true,\\"puts\\":true,\\"pw\\":true,\\"py\\":true,\\"q\\":true,\\"qa\\":true,\\"que\\":true,\\"quite\\":true,\\"qv\\":true,\\"r\\":true,\\"ran\\":true,\\"rather\\":true,\\"rd\\":true,\\"re\\":true,\\"ref\\":true,\\"refs\\":true,\\"regarding\\":true,\\"regardless\\":true,\\"regards\\":true,\\"related\\":true,\\"research\\":true,\\"reserved\\":true,\\"resulted\\":true,\\"resulting\\":true,\\"results\\":true,\\"right\\":true,\\"ring\\":true,\\"ro\\":true,\\"room\\":true,\\"rooms\\":true,\\"round\\":true,\\"ru\\":true,\\"run\\":true,\\"rw\\":true,\\"s\\":true,\\"sa\\":true,\\"said\\":true,\\"same\\":true,\\"saw\\":true,\\"say\\":true,\\"saying\\":true,\\"says\\":true,\\"sb\\":true,\\"sc\\":true,\\"sd\\":true,\\"se\\":true,\\"sec\\":true,\\"seconds\\":true,\\"see\\":true,\\"seeing\\":true,\\"seem\\":true,\\"seemed\\":true,\\"seeming\\":true,\\"seems\\":true,\\"seen\\":true,\\"sees\\":true,\\"self\\":true,\\"selves\\":true,\\"sensible\\":true,\\"sent\\":true,\\"serious\\":true,\\"seven\\":true,\\"seventy\\":true,\\"several\\":true,\\"sg\\":true,\\"sh\\":true,\\"shall\\":true,\\"shan't\\":true,\\"shant\\":true,\\"she\\":true,\\"she'd\\":true,\\"she'll\\":true,\\"she's\\":true,\\"shed\\":true,\\"shell\\":true,\\"shes\\":true,\\"should\\":true,\\"should've\\":true,\\"shouldn\\":true,\\"shouldn't\\":true,\\"shouldnt\\":true,\\"show\\":true,\\"showed\\":true,\\"showing\\":true,\\"shown\\":true,\\"showns\\":true,\\"shows\\":true,\\"si\\":true,\\"side\\":true,\\"sides\\":true,\\"since\\":true,\\"sincere\\":true,\\"site\\":true,\\"six\\":true,\\"sixty\\":true,\\"sj\\":true,\\"sk\\":true,\\"sl\\":true,\\"slightly\\":true,\\"sm\\":true,\\"small\\":true,\\"smaller\\":true,\\"smallest\\":true,\\"sn\\":true,\\"so\\":true,\\"some\\":true,\\"somebody\\":true,\\"someday\\":true,\\"somehow\\":true,\\"someone\\":true,\\"somethan\\":true,\\"something\\":true,\\"sometime\\":true,\\"sometimes\\":true,\\"somewhat\\":true,\\"somewhere\\":true,\\"soon\\":true,\\"sorry\\":true,\\"specified\\":true,\\"specify\\":true,\\"specifying\\":true,\\"sr\\":true,\\"st\\":true,\\"state\\":true,\\"states\\":true,\\"still\\":true,\\"su\\":true,\\"sub\\":true,\\"such\\":true,\\"suggest\\":true,\\"sup\\":true,\\"sure\\":true,\\"sv\\":true,\\"sy\\":true,\\"system\\":true,\\"sz\\":true,\\"t\\":true,\\"t's\\":true,\\"take\\":true,\\"taken\\":true,\\"taking\\":true,\\"tc\\":true,\\"td\\":true,\\"tell\\":true,\\"ten\\":true,\\"tends\\":true,\\"test\\":true,\\"tf\\":true,\\"tg\\":true,\\"th\\":true,\\"than\\":true,\\"thank\\":true,\\"thanks\\":true,\\"thanx\\":true,\\"that\\":true,\\"that'll\\":true,\\"that's\\":true,\\"that've\\":true,\\"thatll\\":true,\\"thats\\":true,\\"thatve\\":true,\\"the\\":true,\\"their\\":true,\\"theirs\\":true,\\"them\\":true,\\"themselves\\":true,\\"then\\":true,\\"thence\\":true,\\"there\\":true,\\"there'd\\":true,\\"there'll\\":true,\\"there're\\":true,\\"there's\\":true,\\"there've\\":true,\\"thereafter\\":true,\\"thereby\\":true,\\"thered\\":true,\\"therefore\\":true,\\"therein\\":true,\\"therell\\":true,\\"thereof\\":true,\\"therere\\":true,\\"theres\\":true,\\"thereto\\":true,\\"thereupon\\":true,\\"thereve\\":true,\\"these\\":true,\\"they\\":true,\\"they'd\\":true,\\"they'll\\":true,\\"they're\\":true,\\"they've\\":true,\\"theyd\\":true,\\"theyll\\":true,\\"theyre\\":true,\\"theyve\\":true,\\"thick\\":true,\\"thin\\":true,\\"thing\\":true,\\"things\\":true,\\"think\\":true,\\"thinks\\":true,\\"third\\":true,\\"thirty\\":true,\\"this\\":true,\\"thorough\\":true,\\"\\":true,\\"those\\":true,\\"thou\\":true,\\"though\\":true,\\"thoughh\\":true,\\"thought\\":true,\\"thoughts\\":true,\\"thousand\\":true,\\"three\\":true,\\"throug\\":true,\\"through\\":true,\\"throughout\\":true,\\"thru\\":true,\\"thus\\":true,\\"til\\":true,\\"till\\":true,\\"tip\\":true,\\"tis\\":true,\\"tj\\":true,\\"tk\\":true,\\"tm\\":true,\\"tn\\":true,\\"to\\":true,\\"today\\":true,\\"together\\":true,\\"too\\":true,\\"took\\":true,\\"top\\":true,\\"toward\\":true,\\"towards\\":true,\\"tp\\":true,\\"tr\\":true,\\"tried\\":true,\\"tries\\":true,\\"trillion\\":true,\\"truly\\":true,\\"try\\":true,\\"trying\\":true,\\"ts\\":true,\\"tt\\":true,\\"turn\\":true,\\"turned\\":true,\\"turning\\":true,\\"turns\\":true,\\"tv\\":true,\\"tw\\":true,\\"twas\\":true,\\"twelve\\":true,\\"twenty\\":true,\\"twice\\":true,\\"two\\":true,\\"tz\\":true,\\"u\\":true,\\"ua\\":true,\\"ug\\":true,\\"uk\\":true,\\"um\\":true,\\"un\\":true,\\"under\\":true,\\"underneath\\":true,\\"undoing\\":true,\\"unless\\":true,\\"unlike\\":true,\\"until\\":true,\\"unto\\":true,\\"up\\":true,\\"upon\\":true,\\"ups\\":true,\\"upwards\\":true,\\"us\\":true,\\"use\\":true,\\"used\\":true,\\"useful\\":true,\\"usefully\\":true,\\"usefulness\\":true,\\"uses\\":true,\\"using\\":true,\\"uucp\\":true,\\"uy\\":true,\\"uz\\":true,\\"v\\":true,\\"va\\":true,\\"value\\":true,\\"various\\":true,\\"vc\\":true,\\"ve\\":true,\\"versus\\":true,\\"very\\":true,\\"vg\\":true,\\"vi\\":true,\\"via\\":true,\\"viz\\":true,\\"vn\\":true,\\"vol\\":true,\\"vols\\":true,\\"vs\\":true,\\"vu\\":true,\\"w\\":true,\\"want\\":true,\\"wanted\\":true,\\"wanting\\":true,\\"wants\\":true,\\"was\\":true,\\"wasn\\":true,\\"wasn't\\":true,\\"wasnt\\":true,\\"way\\":true,\\"ways\\":true,\\"we\\":true,\\"we'd\\":true,\\"we'll\\":true,\\"we're\\":true,\\"we've\\":true,\\"wed\\":true,\\"welcome\\":true,\\"well\\":true,\\"wells\\":true,\\"went\\":true,\\"were\\":true,\\"weren\\":true,\\"weren't\\":true,\\"werent\\":true,\\"weve\\":true,\\"wf\\":true,\\"what\\":true,\\"what'd\\":true,\\"what'll\\":true,\\"what's\\":true,\\"what've\\":true,\\"whatever\\":true,\\"whatll\\":true,\\"whats\\":true,\\"whatve\\":true,\\"when\\":true,\\"when'd\\":true,\\"when'll\\":true,\\"when's\\":true,\\"whence\\":true,\\"whenever\\":true,\\"where\\":true,\\"where'd\\":true,\\"where'll\\":true,\\"where's\\":true,\\"whereafter\\":true,\\"whereas\\":true,\\"whereby\\":true,\\"wherein\\":true,\\"wheres\\":true,\\"whereupon\\":true,\\"wherever\\":true,\\"whether\\":true,\\"which\\":true,\\"whichever\\":true,\\"while\\":true,\\"whilst\\":true,\\"whim\\":true,\\"whither\\":true,\\"who\\":true,\\"who'd\\":true,\\"who'll\\":true,\\"who's\\":true,\\"whod\\":true,\\"whoever\\":true,\\"whole\\":true,\\"wholl\\":true,\\"whom\\":true,\\"whomever\\":true,\\"whos\\":true,\\"whose\\":true,\\"why\\":true,\\"why'd\\":true,\\"why'll\\":true,\\"why's\\":true,\\"widely\\":true,\\"width\\":true,\\"will\\":true,\\"willing\\":true,\\"wish\\":true,\\"with\\":true,\\"within\\":true,\\"without\\":true,\\"won\\":true,\\"won't\\":true,\\"wonder\\":true,\\"wont\\":true,\\"words\\":true,\\"work\\":true,\\"worked\\":true,\\"working\\":true,\\"works\\":true,\\"world\\":true,\\"would\\":true,\\"would've\\":true,\\"wouldn\\":true,\\"wouldn't\\":true,\\"wouldnt\\":true,\\"ws\\":true,\\"www\\":true,\\"x\\":true,\\"y\\":true,\\"ye\\":true,\\"year\\":true,\\"years\\":true,\\"yes\\":true,\\"yet\\":true,\\"you\\":true,\\"you'd\\":true,\\"you'll\\":true,\\"you're\\":true,\\"you've\\":true,\\"youd\\":true,\\"youll\\":true,\\"younger\\":true,\\"youngest\\":true,\\"your\\":true,\\"youre\\":true,\\"yours\\":true,\\"yourself\\":true,\\"yourselves\\":true,\\"youve\\":true,\\"yt\\":true,\\"yu\\":true,\\"z\\":true,\\"za\\":true,\\"zero\\":true,\\"zm\\":true,\\"zr\\":true}"`;
